src/domain/signupGroup/__tests__/validation.test.ts

```diff
@@ -169,6 +186,48 @@ describe('isBelowMaxAge function', () => {
   });
 });
 
+describe('signupSchema function', () => {
+  const validContactPerson: ContactPersonFormFields = {
+    email: 'test@email.com',
+    firstName: 'First name',
+    id: null,
+    lastName: 'Last name',
+    membershipNumber: 'xxx',
+    nativeLanguage: 'fi',
+    notifications: [NOTIFICATIONS.EMAIL],
+    phoneNumber: '0441234567',
+    serviceLanguage: 'fi',
+  };
+
+  test('should return true if contact person is valid', async () => {
+    expect(await testContactPersonSchema(validContactPerson)).toBe(true);
+  });
+
+  const testCases: [Partial<ContactPersonFormFields>][] = [
+    [{ email: '' }],
+    [{ email: 'not-email' }],
```

Comment:
Here too max length test cases for email and phone number could be added.

Reply:
Added test cases for email and phone number max lengths